src/pages/home/report/ReportActionItem.js

```diff
@@ -701,6 +701,13 @@ function ReportActionItem(props) {
     const hasErrors = !_.isEmpty(props.action.errors);
     const whisperedToAccountIDs = props.action.whisperedToAccountIDs || [];
     const isWhisper = whisperedToAccountIDs.length > 0;
+
+    // We currently send whispers to all participants and hide them in the UI.
+    // This is a temporary solution needed for comment-linking.
+    // The long term solution will leverage end-to-end encryption and only targeted users will be able to decrypt.
+    if (isWhisper && ReportActionsUtils.isWhisperActionTargetedToOthers(props.reportAction)) {
```

Comment:
`ReportActionsUtils.isWhisperActionTargetedToOthers` will return false if `isWhisper` is false. So `isWhisper` is unnecessary here.

```suggestion
    if (ReportActionsUtils.isWhisperActionTargetedToOthers(props.reportAction)) {
```

Also, I see above we're using `props.action`, but on this line we're using `props.reportAction`, which seems incorrect.

Reply:
good catch! fixed.